3Sum.py

```diff
@@ -0,0 +1,36 @@
+# Time Complexity : O(n**2)
+# Space Complexity : O(1)
```

Comment:
Do you mean O(n^2)?

Reply:
@neha79091 yes

Reply:
think about the complexity again using algorithm

Reply:
@neha79091 I still think the time complexity is O(n^2)

Reply:
Devashish you are right...its an n2 solution.
@neha79091 Kindly consult me if you are confused about some solution given by a student

Reply:
Yes, your answer is correct, I misunderstood .

Reply:
yup, I misunderstood it a little.